Assets/Scripts/StaminaBar.cs

```diff
@@ -129,7 +129,7 @@ public void FadeIn()
 
     private IEnumerator FadeAlphaCoroutine(float startAlpha, float endAlpha, Image image)

```

Comment:
Maybe also refactor these functions to no longer need a startAlpha?

Reply:
But the start alpha is different depending on Fade in/ fade out

Reply:
Actually true yea, I think this way will work! Merci merci!!!